content/docs/contribution-guide/contributing-guide.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Contributing Overview"
+summary: "How to contribute to Oakestra"
+draft: false
+weight: 401
+toc: true
+seo:
+  title: "" # custom title (optional)
+  description: "" # custom description (recommended)
+  canonical: "" # custom canonical URL (optional)
+  noindex: false # false (default) or true
+---
+
+# Contributing Overview
```

Comment:
```suggestion
```
This title can be removed otw Hugo shows it twice
<img width="512" alt="image" src="https://github.com/user-attachments/assets/1aa15813-9072-49a4-9312-368b9cd68645">